test/topology_custom/test_change_replication_factor_1_to_0_with_vnodes.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import logging
+import pytest
+import asyncio
+import time
+
+from cassandra import ConsistencyLevel  # type: ignore
+from cassandra.query import SimpleStatement  # type: ignore
+from test.pylib.manager_client import ManagerClient
+from test.pylib.util import wait_for_cql_and_get_hosts
+
+
+logger = logging.getLogger(__name__)
+CONFIG = {"endpoint_snitch": "GossipingPropertyFileSnitch", "enable_tablets": False}
```

Comment:
are the tests identical except for the tablets setting? if so, would it make sense to make a single parametrized test?

Reply:
Yes, the tests are identical except for the setting `enable_tablets`.  are you suggesting moving the duplicated part to a separate file and use it in 2 files? or to join everything to one file? If so, how can we run a single test, eg for vnodes only?

Reply:
I was thinking more of just having a single file with a single parametrized test. You can then specify the parameter as well when running the test (looks like "`test_change_replication_factor_1_to_0[True]`" or `False` might work).

Or even better, it seems that you can actually assign an ID to the parameter value, like shown e.g. here:  https://stackoverflow.com/a/75109059

So you could do maybe something like:
```
    @pytest.mark.parametrize(
        "use_tablets",
        [
            pytest.param(True, id="tablets"),
            pytest.param(False, id="vnodes"),
        ],
    )
```

and then select like e.g. "`test_change_replication_factor_1_to_0[vnodes]`"

Reply:
I'll try it